src/containers/Users.js

```diff
@@ -85,6 +85,12 @@ class Users extends React.Component {
 			});
 	}
 
+	componentDidUpdate() {
+		const element = document.getElementById("user-details");
+		
+		element.scrollIntoView({behavior: 'smooth'});
+	}
```

Comment:
I don't think putting this in componentDidUpdate() is a good idea. This will also run when you click "search", which is problematic

Reply:
let's put it in the handleFormSubmit() function

Reply:
I just wanna test the lifecycle method lul. Sorry